server/controllers/async-controllers.js

```diff
@@ -0,0 +1,116 @@
+import { getSeries } from '../services/wordpress';
+import { PromoListFactory } from '../model/promo-list';
+import { getForwardFill, getUnpublishedSeries } from '../model/series';
+import { getSeriesColor } from '../data/series';
+import { createNumberedList } from '../model/numbered-list';
+import { getLatestTweets } from '../services/twitter';
+import { getLatestInstagramPosts } from '../services/instagram';
+
+export const seriesNav = async(ctx, next) => {
+  const {id} = ctx.params;
+  const {current} = ctx.request.query;
+  const seriesResponse = await getSeries(id, 6, 1);
+  const series = seriesResponse ? getForwardFill(seriesResponse) : getUnpublishedSeries(id);
+
+  const color = getSeriesColor(id);
+  const promoList = PromoListFactory.fromSeries(series);
+  const items = promoList.items.toJS();
+  const image = items[0].image;
+  const seriesNavModel = createNumberedList({
+    name: promoList.name,
+    image: image,
+    items: items,
+    color: color
+  });
+
+  ctx.render('components/numbered-list/index', {
+    current,
+    model: seriesNavModel,
+    modifiers: ['horizontal', 'sticky'],
+    data: {
+      classes: ['js-series-nav'],
+      sliderId: `series-nav--${id}`
+    }
+  });
+
+  ctx.body = {
+    html: ctx.body
+  };
+
+  return next();
+};
+
+export const seriesTransporter = async(ctx, next) => {
+  const { id } = ctx.params;
+  const { current } = ctx.request.query;
+  const seriesResponse = await getSeries(id, 6, {page: 1});
+  const series = seriesResponse ? getForwardFill(seriesResponse) : getUnpublishedSeries(id);
+
+  const color = getSeriesColor(id);
+  const promoList = PromoListFactory.fromSeries(series);
+  const items = promoList.items.toJS();
+  const image = items[0].image;
+  const seriesNavModel = createNumberedList({
+    name: promoList.name,
+    image: image,
+    items: items,
+    color: color
+  });
+
+  ctx.render('components/numbered-list/index', {
+    current,
+    model: seriesNavModel,
+    modifiers: ['transporter'],
+    data: {
+      classes: ['js-numbered-list-transporter'],
+      sliderId: `transporter--${id}`
+    }
+  });
+
+  ctx.body = {
+    html: ctx.body
+  };
+
+  return next();
+};
+
+export const latestInstagramPosts = async(ctx, next) => {
+  const instagramPosts = await getLatestInstagramPosts(10);
+
+  ctx.render('components/social-media-block/index', {
+    model: {
+      posts: instagramPosts,
+      service: 'Instagram',
+      icon: 'social/instagram',
+      url: 'https://instagram.com/wellcomecollection',
+      handle: 'wellcomecollection'
+    }
+  });
+
+  ctx.body = {
+    html: ctx.body
+  };
+
+  return next();
+};
+
+export const latestTweets = async(ctx, next) => {
+  const count = ctx.params.count || 4;
```

Comment:
Indeed - let's add things when we need them - the code base is getting bigger, and more complicated.

If I saw this, I would assume we're using it, so not change it if I needed and potentially work around it, making refactoring harder to fathom.